File: Level-1/49.js
function solution(left, right) {
    let answer = 0;
    
    for(let i = left; i <= right; i++) {
        let set = new Set();
        for(let j = 1; j <= Math.sqrt(i); j++) {
            if(i % j === 0) {
                set.add(j);
                set.add(i / j);
            }
        }
        set.size % 2 === 0 ? answer += i : answer -= i;
    }
    
    return answer;
}

File: Level-0/49.java
class Solution {
    public int solution(String my_string) {
        int answer = 0;
        for(int i = 0; i < my_string.length(); i++){
            if(my_string.charAt(i) >= 49 && my_string.charAt(i) <=57){
                answer += my_string.charAt(i) - '0';
            }
        }
        return answer;
    }
}
